### PR TITLE
Fix selinux ctx setup in image generator

### PR DESCRIPTION
**Pull request by MykolaSolyanko**

### CODE REVIEW

**Review by mykola-kobets-epam — APPROVED** [review]

File: classes/rootfs-image-generator.bbclass [hunk 1]
```diff
@@ -72,14 +72,24 @@ create_incremental_update() {
...
         ${@'' if not d.getVar('AOS_ROOTFS_EXCLUDE_ITEMS') else '-e '+ d.getVar('AOS_ROOTFS_EXCLUDE_ITEMS')}
 }
 
 set_selinux_context() {
-    ROOTFS_FULL_PATH=$(realpath ${AOS_ROOTFS_SOURCE_DIR})
-    setfiles -m -r ${ROOTFS_FULL_PATH} ${ROOTFS_FULL_PATH}/etc/selinux/aos/contexts/files/file_contexts ${ROOTFS_FULL_PATH}
+    target_dir="$1"
+    rootfs_dir="$2"
+
+    TARGET_DIR_FULL_PATH=$(realpath "$target_dir")
```
Comment (al1img) [inline classes/rootfs-image-generator.bbclass:88]: Looks like these vars are local. Use small letters to name them.

File: classes/rootfs-image-generator.bbclass [hunk 2]
```diff
@@ -89,7 +99,7 @@ fakeroot do_create_rootfs_image() {
     fi
 
     if ${@bb.utils.contains('DISTRO_FEATURES', 'selinux', 'true', 'false', d)}; then
-        set_selinux_context
+        set_selinux_context "${AOS_ROOTFS_SOURCE_DIR}" "${AOS_ROOTFS_SOURCE_DIR}"
```
[the remark this answers is not in the document]
Comment (MykolaSolyanko) [reply classes/rootfs-image-generator.bbclass:102]: yes

**Review by mlohvynenko — APPROVED** [review]

File: classes/rootfs-image-generator.bbclass (hunk 2 above)
[the remark this answers is not in the document]
Comment (al1img) [reply classes/rootfs-image-generator.bbclass:102]: Why don't have only one parameter in this case?

File: classes/rootfs-image-generator.bbclass (hunk 1 above)
Comment (MykolaSolyanko) [reply classes/rootfs-image-generator.bbclass]: Done

**Review by al1img — APPROVED** [review]

**Pull request merged**